terraform/gcp/projects/cluster.tfvars.template

```diff
@@ -81,8 +81,8 @@ notebook_nodes = {
   }
 }
 
-dask_nodes = {
 {% if hub_type == "daskhub" %}
+dask_nodes = {
```

Comment:
Like this, the logic below may reference an uninitilaized variable - making the logic below possibly more convoluted to handle it. Maybe its cleaner if we instead always initialize it?

Reply:
@consideRatio, I've updated the deployer command so that the hub_type will be requested as a prompt if no provided at all, where the default is `basehub`. I don't believe it can ever be uninitialized. It will look like this:

<img width="1440" alt="Screenshot 2024-04-03 at 16 52 36" src="https://github.com/2i2c-org/infrastructure/assets/7579677/4c8aa560-b2e0-44db-85b8-7557b057d696">